### PR TITLE
v0.7.21 merge from dev to main

### PR DESCRIPTION
**Pull request by chris18890**
mostly small tpyo/spacing fixes in the code rqhter than any user-facing changes

### CODE REVIEW

**Comment by chris18890** [comment]
@elplatt / @ramgarden can I get a review on this PR?

File: crm/include/page.inc.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ function core_page (&$page_data, $page_name, $options) {
     $latestNews = '<p>Welcome to ' . title() . ' version ' . crm_version() . '!</p>';
     // Modify this variable with valid HTML between the apostrophes to display update text to users on login
     $latestNews = $latestNews . '
-        <p><p>
+        <p></p>
```
Comment (ramgarden) [inline crm/include/page.inc.php:60]: was this actually supposed to be two empty paragraphs like a triple space?  <p/><p/>

File: crm/include/sys/module.inc.php [hunk 2]
```diff
@@ -299,7 +299,6 @@ function module_upgrade_form () {
  * @return The url to redirect to on completion.
  */
 function command_module_install () {
-    global $db_connect;
```
Comment (ramgarden) [inline crm/include/sys/module.inc.php:302]: Why don't we need the db connect to be global here?

File: crm/modules/member/install.inc.php [hunk 3]
```diff
@@ -28,7 +28,7 @@
 function member_install($old_revision = 0) {
     global $db_connect;
     if ($old_revision == 1) {
-        error_log('The database version is too old to upgrade to this release of ' . title(). '.  Please upgrade one release at a time.');
+        error_log('The database version is too old to upgrade to this release of ' . title() . '. Please upgrade one release at a time.');
```
Comment (ramgarden) [inline crm/modules/member/install.inc.php:31]: Woo! that's some confusing stuff right there.  Which dot is concatenate and which dot is the actual period in the sentence!?  Good catch here.

File: crm/modules/user_meta/user_meta.inc.php [hunk 4]
```diff
@@ -434,11 +434,13 @@ function user_meta_cross_table ($opts) {
...
                 // insert new tag in new row at a fixed offset.
-                for ( $i = 1 ; $i < $count+1; $i++) {
-                    if ( $table['columns'][$i]['title'] == $user_meta['tagstr'] ) {
+                for ($i = 1 ; $i < $count+1; $i++) {
+                    if ($table['columns'][$i]['title'] == $user_meta['tagstr']) {
                         $row[$i] = '<input type="checkbox" name="'.$user_meta['tagstr'].'" value="1" checked="checked" disabled=true/>';
                     } else {
-                        if (!array_key_exists($i, $row) ) { $row[$i] = '';}
+                        if (!array_key_exists($i, $row)) {
+                            $row[$i] = '';
```
Comment (ramgarden) [inline crm/modules/user_meta/user_meta.inc.php:442]: much more readable and maintainable this way!

**Review by ramgarden — APPROVED** [review]
Approved with comments.  Check those before you merge.

**Pull request merged**